apps/monarch/monarch-frontend/src/app/SearchTherapists.tsx

```diff
@@ -347,7 +368,7 @@ const EditButton: React.FC<{
                 <NumberInput
                   value={familiesHelped}
                   onChange={(valueString, valueNumber) =>
-                    setFamiliesHelped(valueNumber)
+                    setFamiliesHelped(valueNumber || 0)
```

Comment:
adding an `|| 0` since if someone presses backspace/delete with one digit, we get `NaN` in the textbox